### ISSUE TITLE
"Modal" mask active after file upload in file manager

### ISSUE THREAD

**Issue by eschrip** [qooob/authentic-theme#1006]
using Firefox 58.0.1 (64-bit) on Mac.

After I upload a file using file manager, the gray mask used to prevent UI under the dialog fails to removed after upload is complete.   The upload completes, and the display is updated, but the gray transparent mask remains, and I have to refresh the browser to gain access.

**Comment by qooob** [comment]
Hi,

What theme version do you use?

**Comment by eschrip** [comment]
Of course I forgot the key information.. sorry about that.

Authentic Theme 19.04

Eric

**Comment by eschrip** [comment]
Strangely enough I “thought” I had done that already just yesterday… I had to install “git” on the machine to do it.   But now I don’t seem to have a way to force it except to download.

However, when I attempt to install the19.09 download, it is insisting on 1.872 of webmin.  But I am on 1.870 , and it is telling me that is the latest for me.   The webmin site seems to reflect this as well.

So at this point, not sure where that leaves me.   Not a critical issue, since I can just handle the problem fairly easy.

Eric

**Comment by qooob** [comment]
It's okay to upgrade to 19.09. There is small incompatibility. It will work.

**Comment by eschrip** [comment]
The only issue is that it won’t let me.  As it is insisting on a version of Webmin that I can’t get.

Eric

**Comment by eschrip** [comment]
Just an FYI,

I installed the 19.08 (Which appears to have the new upload function) and it installs without any problems.

I will keep my eyes open for any other changes, and upgrade as soon as versions sync up.

Eric

**Comment by eschrip** [comment]
One last followup.. sorry to bother… I tried from the command line (which is probably what you were assuming anyway).. and the install is fine.

Sorry for excess.

Thanks.
Eric

**Comment by eschrip** [comment]
And because I noticed... I NOW have a Force Update to the theme.  Prior to this I did not.

**Comment by qooob** [comment]
So everything is fine and working?

**Comment by eschrip** [comment]
Yes thank you.  Sorry for the flood of messages.  All is working.

Eric
